site/content/docs/getting-started/pre-requisites.md

```diff
@@ -22,7 +22,7 @@ dotnet --version
 
 #### Node.js
 
-* The deployment tool requires the [AWS Cloud Development Kit (CDK)](https://docs.aws.amazon.com/cdk/latest/guide/), and the AWS CDK requires [Node.js](https://nodejs.org/en/download/). AWS CDK requires Node.js, versions 14.x, 16.x, 18.x (or later) - we recommend installing the latest LTS version.
+* The deployment tool requires the [AWS Cloud Development Kit (CDK)](https://docs.aws.amazon.com/cdk/latest/guide/), and the AWS CDK requires [Node.js](https://nodejs.org/en/download/). AWS CDK requires Node.js 18.x (or later) - we recommend installing the latest LTS version.

```

Comment:
nit: Does CDK require 18 now? They still say 14 here: https://github.com/aws/aws-cdk#getting-started

Fine if _we_ require it, just don't want to mislead anybody.

Reply:
I updated the wording to not explicitly indicate that CDK is requiring a specific version, rather a specific version is required to use the tool.